### PR TITLE
Added two egg facts to facts.txt

### PR DESCRIPTION
**Pull request by ChristofferNissen**

### CODE REVIEW

File: facts.txt [hunk 1]
```diff
@@ -141,4 +141,8 @@
         "The average age of a French oak tree harvested for use in creating wine barrels is 170 years."
         "10,000 varieties of wine grapes exist worldwide."
     },
+	"Egg": {
```
Comment (rahuldecoded) [inline facts.txt]: comma is missing in every line.

File: facts.txt [hunk 2]
```diff
@@ -141,4 +141,8 @@
...
+		"The color of an egg yolk is determined by a hen’s diet. If you crack open your egg to discover a dark yellow yolk, the hen was probably fed green vegetables. A medium-yellow yolk would indicate a diet of corn and alfalfa while a light-yellow yolk could be the result of eating wheat and barley."
+		"Harriet the hen laid the world’s largest egg in 2010, measuring a hefty 9.1 inches in diameter."
+	}
```
Comment (rahuldecoded) [inline facts.txt]: comma missing after the braces. Didn't you read README before raising a PR?

**Comment by ChristofferNissen** [comment]
Yes, look at other prs. Just a mistake. I will fix it.

File: facts.txt (hunk 1 above)
Comment (ChristofferNissen) [reply facts.txt]: Are you talking about within the curly braces? I am just following the convention i see in the file.

**Pull request closed without merging**